bench-streamer/src/main.rs

```diff
@@ -20,7 +21,7 @@ use {
 };
 
 fn producer(addr: &SocketAddr, exit: Arc<AtomicBool>) -> JoinHandle<()> {
-    let send = UdpSocket::bind("0.0.0.0:0").unwrap();
+    let send = bind_to(IpAddr::V4(Ipv4Addr::UNSPECIFIED), 0, false).unwrap();
```

Comment:
Looking at the green lines in the diff, I see either of these repeated ALOT:
```rust
bind_to(IpAddr::V4(Ipv4Addr::LOCALHOST), 0, false).unwrap();
bind_to(IpAddr::V4(Ipv4Addr::UNSPECIFIED), 0, false).unwrap();
```
Generally, comments for the params are nice:
```
bind_to(IpAddr::V4(Ipv4Addr::LOCALHOST), /*port:*/ 0, /*reuseport:*/ false).unwrap();
```
But given how often the two are repeated, I actually think we should add some helpers for these "standard" cases:
```rust
fn bind_to_localhost() -> io::Result<UdpSocket> {
    bind_to(IpAddr::V4(Ipv4Addr::LOCALHOST), /*port:*/ 0, /*reuseport:*/ false)
}

fn bind_to_unspecified() -> io::Result<UdpSocket> {
    bind_to(IpAddr::V4(Ipv4Addr::UNSPECIFIED), /*port:*/ 0, /*reuseport:*/ false)
}
```
I'm not sure if there are any production (non test/bench) instances where we use `LOCALHOST` or `UNSPECIFIED`, but if there are not, it'd also be nice to limit these new functions under a `#[cfg(test)]` or dev-context-only-utils

Reply:
good idea. `bind_to_unspecified` is not test only. It is used in `ip_echo_server` but `bind_to_async`, `bind_to_localhost`, and `bind_to_localhost_async` can be `#[cfg(feature = "dev-context-only-utils")]`